indico/modules/events/management/controllers.py

```diff
@@ -178,3 +178,13 @@ def _update_session_coordinator_privs(self, form):
             log_msg = 'Session coordinator privilege changed to {}: {}'.format(form.data[priv_field],
                                                                                COORDINATOR_PRIV_TITLES[priv_field])
             self.event_new.log(EventLogRealm.management, EventLogKind.positive, 'Protection', log_msg, session.user)
+
+
+class RHMoveEvent(RHConferenceModifBase):
```

Comment:
It's because of the last part of your message that I believe it would better fit in category management :)